src/views/components/shared/Input/component.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Form, Input as AntInput } from 'antd'
+import { EyeTwoTone, EyeInvisibleOutlined } from '@ant-design/icons'
+
+import { propTypes, defaultProps } from './types'
+
+export const iconRender = (visible) =>
+  visible ? <EyeTwoTone /> : <EyeInvisibleOutlined />
+
+const Input = ({
+  placeholder,
+  validateStatus,
+  validationMessage,
+  type,
+  value,
+  onChange,
+  onBlur,
+  name,
+  prefix
+}) => (
+  <Form.Item
+    validateStatus={validateStatus}
+    help={validationMessage}
+  >
+    {type === 'password' ? (
```

Comment:
not sure about implementation, looks like `PasswordInput` and `Input` are different components

Reply:
Fixed